assignments/arrays.js

```diff
@@ -63,14 +63,20 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let inventorySearchResult = inventory.filter(function (vehicle) {
```

Comment:
Great work with this filter. I appreciate that you solution makes no assumptions about the index in the array of the car with the id of 33. 💯